server.go

```diff
@@ -34,6 +36,9 @@ import (
 // It must be a var so it can be set at link time.
 var VERSION = "dirty"
 
+// REDACTED should be a constant since we use it enough.
```

Comment:
Can we replace this constant with an explanation of what it refers to (ie, so that we don't accidentally dump secrets if we log the config).